evinser.js

```diff
@@ -335,7 +335,8 @@ export const initFromSbom = (components, language) => {
  */
 export const analyzeProject = async (dbObjMap, options) => {
   const dirPath = options._[0] || ".";
-  const language = options.language;
+  const languages = options.language;
+  const language = Array.isArray(languages) ? languages[0] : languages;
```

Comment:
We need a separate branch to add proper multi-type support for evinse. Essentially, we need to run the evinser multiple times per language type.